window/src/os/windows/window.rs

```diff
@@ -989,10 +1241,12 @@ unsafe fn wm_kill_focus(
     _lparam: LPARAM,
 ) -> Option<LRESULT> {
     if let Some(inner) = rc_from_hwnd(hwnd) {
+        let mut inner = inner.borrow_mut();
+        inner.events.dispatch(WindowEvent::FocusChanged(false));
+        let appearance = inner.appearance;
         inner
-            .borrow_mut()
             .events
-            .dispatch(WindowEvent::FocusChanged(false));
+            .dispatch(WindowEvent::AppearanceChanged(appearance));
```

Comment:
This causes the config to be reloaded each time the window focus is changed, which results in a noticeable delay in responsiveness when focusing the window and for the bottom window border color to lag behind the others.

To resolve this:

* Remove the `AppearanceChanged` dispatch from here and from `wm_set_focus` above
* Add a call to fetch the OS parameters directly in `TermWindow::focus_changed`